doc/api/esm.md

```diff
@@ -180,83 +180,78 @@ unspecified.
 There are two fields that can define entry points for a package: `"main"` and
 `"exports"`. The `"main"` field is supported in all versions of Node.js, but its
 capabilities are limited: it only defines the main entry point of the package.
-The `"exports"` field, part of [Package Exports][], provides an alternative to
-`"main"` where the package main entry point can be defined while also
-encapsulating the package, preventing any other entry points besides those
-defined in `"exports"`. If package entry points are defined in both `"main"` and
-`"exports"`, the latter takes precedence in versions of Node.js that support
-`"exports"`. [Conditional Exports][] can also be used within `"exports"` to
-define different package entry points per environment.
 
-#### `package.json` `"main"`
+The `"exports"` field provides an alternative to `"main"` where the package
+main entry point can be defined while also encapsulating the package, preventing
+any other entry points besides those defined in `"exports"`. If package entry
+points are defined in both `"main"` and `"exports"`, the latter takes precedence
+in versions of Node.js that support `"exports"`. [Conditional Exports][] can
+also be used within `"exports"` to define different package entry points per
+environment.
 
-The `package.json` `"main"` field defines the entry point for a package,
-whether the package is included into CommonJS via `require` or into an ES
-module via `import`.
+When using both `import` and `require` the `"exports"` field will be resolved
+if it is set, otherwise the `"main"` field will be resolved.
+
+When setting any package entry points to ES modules, `require()` will not work in
+both modern and legacy Node.js versions, so it is important to carefully follow
+[the dual CommonJS/ES module packages section][] to properly handle these types
+of backwards-compatible ES module support.
```

Comment:
```suggestion
Package entry points defined via `"exports"` apply to both `import` and `require`;
`"exports"` is **not** specific to ES modules. If both `"exports"` and `"main"` are
defined, the `"exports"` field takes precedence over `"main"`.

This is important with regard to `require`, since `require` of ES module files
throws an error in all versions of Node.js. To create a package that works both
in modern Node.js via `import` and in legacy Node.js versions via `require`, see
[the dual CommonJS/ES module packages section][].
```

Reply:
I wanted to specifically note that "main" also applies with this, as the section noted previously.

How about - 

```suggestion
All package entry points apply to both `import` and `require`; `"exports"` and
`"main"` are **not** specific to ES modules or CommonJS. If both `"exports"`
and `"main"` are defined, the `"exports"` field takes precedence over `"main"`.

This is important with regard to `require`, since `require` of ES module files
throws an error in all versions of Node.js. To create a package that works both
in modern Node.js via `import` and in legacy Node.js versions via `require`, see
[the dual CommonJS/ES module packages section][].
```

Reply:
Added with rewording.